src/PublicApi/Dockerfile

```diff
@@ -1,11 +1,11 @@
 #See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-buster-slim AS base
+FROM mcr.microsoft.com/dotnet/aspnet:5.0.0-buster-slim AS base
```

Comment:
Use 5.0 and avoid minor versions.

Reply:
Removed minor versions from both the docker files.

Reply:
Removed minor versions